modules/ROOT/pages/forum-moderation.adoc

```diff
@@ -4,6 +4,78 @@ include::_attributes.adoc[]
 
-Delete, approve for publication, other features such as locking user accounts.
+Developers can use the API Community Manager forums to share tips and advice on using the organization's APIs.
 
-Link to Salesforce documentation for further info.
+Your organization's community operator can moderate the forums. This includes replying to new discussions and questions, reviewing audit trails, and creating and managing forum rules. These rules can filter posts, other content, and members based on criteria defined by the operator. Members matching the criteria can be frozen, and posts and other content matching the criteria can be flagged, blocked, automatically edited to remove banned terms, or held for the operator to review them.
+
+// Will the pre-made community in the workshop have various posts, including content that matches various criteria to be held for review, and users that were frozen for posting too often or similar criteria?
+
+To moderate forums, log in, select a community, and select *Moderation*.
+
+The *Home* tab shows these options:
+
+* *Pending Discussions*: Respond to and moderate new questions and discussions.
+* *Flagged Files*: Unflag or delete files flagged as suspicious.
+* *Flagged Direct Messages*: Unflag or delete direct messages flagged as suspicious.
+* *Flagged Private Messages*: Unflag or delete private messages flagged as suspicious.
+* *Frozen Members*: Unfreeze user accounts.
+
+The *Moderate* tab shows these options:
+
+* *Pending Discussions*: Respond to and moderate new questions and discussions.
+* *Flagged Files*: Unflag or delete files flagged as suspicious.
+* *Flagged Messages*: Unflag or delete messages flagged as suspicious.
+* *Frozen Members*: Unfreeze user accounts.
+* *Audit Trail*: Review an audit trail of all user activity on the forums.
+
+The *Rules* tab shows these options:
+
+* *Moderation Rules*: View, add, and edit rules to moderate member-generated content. For example, members who post or comment too frequently can be frozen, the first post or comment made by a new member can be held for moderator review, and posts or comments with banned keywords can be blocked, flagged, held for review, or edited so that the banned words are replaced with asterisks.
+* *Content Criteria*: Filter content with criteria such as lists of keywords to block. For example, you might have one list of terms found in financial scam posts, and another list of terms used as offensive insults.
+* *Member Criteria*: Filter members with criteria such as user creation date, whether the user has contributed to the community before, and user profiles such as customers and partners.
+
+== Moderating with Community Workspaces
+
+Community administrators may either use the same forum pages that are seen by external users to administer and post on the forums, or use Community Workspaces pages in Salesforce to administer and post on the forums and to access many other options for monitoring and managing communities.
```

Comment:
Avoid using the word "may"--it is ambiguous in that it can mean they have permission to do a thing, or that they have a choice about doing a thing. 
Community administrators can either use...

Reply:
Much better! Done.